Apps/Playground/Scripts/experience.js

```diff
@@ -177,6 +178,73 @@ CreateBoxAsync().then(function () {
                     }, 5000);
                 }
 
+                // Showing visualization for ARKit LiDAR mesh data
+                if(meshDetection) {
+                    var mat = new BABYLON.StandardMaterial("mat", scene);
+                    mat.wireframe = true;
+                    mat.diffuseColor = BABYLON.Color3.Blue();
+                    const xrMeshes = xr.baseExperience.featuresManager.enableFeature(
+                    BABYLON.WebXRFeatureName.MESH_DETECTION,
+                    "latest",
+                    {convertCoordinateSystems: true});
+                    console.log("Enabled mesh detection.");
+                    const meshes = [];
+                    const meshID = [];
+                    
```

Comment:
nit: we should probably go through and remove all unused spaces on empty lines. Some editors can have their preferences set to do this by default.

FWIW, its not that the spaces do anything substantial to the code or impact the application, but adding empty characters like this has historically caused weird empty space merge conflicts for folks continuing to iterate on the code.